docs/user/technical-reference/04-20-internal-registry.md

```diff
@@ -0,0 +1,18 @@
+# Internal Docker Registry
+
+The Docker Registry module comes with the internal Docker registry, which stores the Function container images without using the third-party registry.
+
+The internal Docker registry is not recommended for production, as it's not deployed in the High Availability (HA) setup and has limited storage space and no garbage collection of the orphaned images.
+
+Still, it is very convenient for development and getting first-time experience with the Kyma Docker Registry.
+
+1. Push Function image to the Docker registry using the in-cluster URL.
```

Comment:
What are these steps? Is it something a user must do? For what purpose?
If it's only a list of the Docker Registry features, please use unordered list.